metadata-service/auth-servlet-impl/src/main/java/com/datahub/auth/authentication/AuthServiceController.java

```diff
@@ -123,9 +123,7 @@ CompletableFuture<ResponseEntity<String>> generateSessionTokenForUser(
     try {
       bodyJson = mapper.readTree(jsonStr);
     } catch (JsonProcessingException e) {
-      log.error(
-          String.format(
-              "Failed to parse json while attempting to generate session token %s", jsonStr));
+      log.error("Failed to parse json while attempting to generate session token {}", jsonStr, e);
```

Comment:
Any chance this exposes a token?

Reply:
The request JSON only contains the user id: https://github.com/acryldata/datahub/blob/8f6d7b44308590bbb47e04d5f06b5735d68c28c9/datahub-frontend/app/client/AuthServiceClient.java#L89-L95